bucket/butterflow.json

```diff
@@ -6,7 +6,7 @@
     "architecture": {
         "64bit": {
             "url": "https://github.com/dthpham/butterflow/releases/download/0.2.4.alpha2/butterflow-0.2.4a4-win64.zip",
-            "hash": "856ad05bafaeb9d084d6b1cbf0bdec17defeef438e610f6a3a772087acea301b",
+            "hash": "a77fbbdbdd0d85bb31feac30e53378a36ff4fb2a8a98ba15a6fa06def4b36ad1",
```

Comment:
Wrong hash

Reply:
I got the right hash for 0.2.4.alpha2. `a77fbbdbdd0d85bb31feac30e53378a36ff4fb2a8a98ba15a6fa06def4b36ad1` is for 0.2.3.